mtmount/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# This docker file cannot fully automate the EUI installation. You must carry on
+# with the installation of the VIPM package installs to complete the installation. 
+
+FROM centos:centos7
+
+LABEL maintainer Andrew Heyer <aheyer@lsst.org>
+
+ENV USER ${USER:-saluser}
+ENV HOME /home/saluser
+USER root
+# Create user and group
+RUN groupadd lsst && adduser -u 1004 -m -g lsst -s /bin/bash saluser
+
+COPY lsst-ts-nexus.repo /etc/yum.repos.d/lsst-ts-nexus.repo
+COPY wandisco-git.repo /etc/yum.repos.d/wandisco-git.repo
+RUN rpm --import http://opensource.wandisco.com/RPM-GPG-KEY-WANdisco
+RUN yum install -y git wget
```

Comment:
before playing with yum,it's better to update it:
```bash
yum -y update
```
There is also curl, you don't need wget then (but see below). curl -o <file> <url>

Reply:
I tried doing an update, however I keep getting an error that I haven't been able to resolve. -> Processing Dependency: /usr/bin/python for package: python-chardet-2.2.1-3.el7.noarch
--> Processing Dependency: /usr/bin/python for package: yum-utils-1.1.31-54.el7_8.noarch
--> Finished Dependency Resolution
Error: Package: yum-utils-1.1.31-54.el7_8.noarch (updates)
           Requires: /usr/bin/python
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               Not found
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
Error: Package: python-chardet-2.2.1-3.el7.noarch (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: rpm-python-4.11.3-43.el7.x86_64 (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: yum-3.4.3-167.el7.centos.noarch (@CentOS)
           Requires: /usr/bin/python
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               Not found
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
Error: Package: python-urlgrabber-3.10-10.el7.noarch (@CentOS)
           Requires: /usr/bin/python
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               Not found
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
Error: Package: yum-metadata-parser-1.1.4-10.el7.x86_64 (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: python-gobject-base-3.22.0-1.el7_4.1.x86_64 (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: pygpgme-0.3-9.el7.x86_64 (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: python-chardet-2.2.1-3.el7.noarch (@CentOS)
           Requires: /usr/bin/python
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               Not found
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
Error: Package: pyliblzma-0.5.3-11.el7.x86_64 (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: pyxattr-0.5.1-5.el7.x86_64 (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: python-urlgrabber-3.10-10.el7.noarch (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: python-iniparse-0.4-9.el7.noarch (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: yum-utils-1.1.31-54.el7_8.noarch (updates)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: yum-3.4.3-167.el7.centos.noarch (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: dbus-python-1.1.1-9.el7.x86_64 (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
 You could try using --skip-broken to work around the problem
Error: Package: yum-3.4.3-167.el7.centos.noarch (@CentOS)
           Requires: python-sqlite
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python-sqlite = 2.3.2
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
Error: Package: libxml2-python-2.9.1-6.el7.4.x86_64 (@CentOS)
           Requires: /usr/bin/python
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               Not found
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
Error: Package: python-pycurl-7.19.0-19.el7.x86_64 (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: python-kitchen-1.1.1-5.el7.noarch (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
Error: Package: libxml2-python-2.9.1-6.el7.4.x86_64 (@CentOS)
           Requires: python(abi) = 2.7
           Removing: python-2.7.5-88.el7.x86_64 (@CentOS)
               python(abi) = 2.7
               python(abi) = 2.7
           Updated By: python-3.7.3-1.el7.centos.x86_64 (lsst-ts)
               Not found
           Available: python3-3.6.8-13.el7.i686 (base)
               python(abi) = 3.6
               python(abi) = 3.6
 You could try running: rpm -Va --nofiles --nodigest
The command '/bin/sh -c yum -y update' returned a non-zero code: 1